src/test/kotlin/org/taymyr/lagom/elasticsearch/AggregationsIT.kt

```diff
@@ -185,6 +185,66 @@ class AggregationsIT : WordSpec() {
                     category.categoryId shouldBe 200
                 }
             }
+            "successfully prevented adding documents with fields not specified in index" {
```

Comment:
* Move test to `ElasticIndicesIT`.  
* Need test for all values of dynamic type